start.php

```diff
@@ -5,5 +5,13 @@
 function xoxco_tags_init() {
 	elgg_extend_view('css/elgg', 'xoxco_tags/css/elgg_after', 501);
 	elgg_extend_view('css/admin', 'xoxco_tags/css/elgg_after', 501);
-	elgg_extend_view('js/elgg', 'xoxco_tags/js/elgg_after', 501);
-}
+
+	elgg_define_js('jquery.tagsinput', array(
+		'src' => '/mod/xoxco_tags/js/lib/jquery.tagsinput.min.js',
+	));
+
+	// Include the feature only if Elgg is displaying the tags input
+	elgg_register_plugin_hook_handler('view', 'input/tags', function () {
+		elgg_require_js('xoxco_tags');
```

Comment:
I'd also like to manually require from an inline script, for forms loaded by ajax.